libbpf-tools/memleak.c

```diff
@@ -0,0 +1,707 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+// Copyright (c) 2022 Jackie Dinh
+//
+// Based on memleak(8) from BCC by Sasha Goldshtein
+#include <argp.h>
+#include <errno.h>
+#include <signal.h>
+#include <stdio.h>
+#include <sys/resource.h>
+#include <time.h>
+#include <unistd.h>
+
+#include <bpf/libbpf.h>
+#include <bpf/bpf.h>
+
+#include "memleak.h"
+#include "memleak.skel.h"
+#include "trace_helpers.h"
+#include "uprobe_helpers.h"
+
+#define PATH_MAX	4096
```

Comment:
Please include proper header file for this.

Reply:
not sure I understanding it corretly. Can you be more specific on it?